src/projection.jl

```diff
@@ -255,6 +255,29 @@ end
 ##### `Base`, part II: return of the Tangent
 #####
 
+# Tuple
+function ProjectTo(xs::Tuple)
+    elements = map(xs) do x
+        x isa Union{Number, AbstractArray{<:Number}} ? ProjectTo(x) : identity
```

Comment:
Also, this accepts `Tuple{Any}`, should it? 

Don't think you can dispatch on eltype to avoid this, excetps when type is homogeneous.